packages/manager/modules/billing/src/orders/purchaseOrders/new-purchase/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,18 @@
+{
+  "purchaseOrders_add_page_title": "Créer une référence interne",
+  "purchaseOrders_form_add_internal_reference": "Créer votre référence interne",
+  "purchaseOrders_form_add_purchase_order": "Créer votre purchase order",
+  "purchaseOrders_form_add_internal_reference_label": "Nom de la référence interne",
+  "purchaseOrders_form_add_internal_reference_placeholder": "Référence interne",
+  "purchaseOrders_form_add_internal_reference_sub_info": "La référence ne doit pas excéder 20 caractères",
+  "purchaseOrders_form_add_purchase_order_label": "Nom du purchase order",
+  "purchaseOrders_form_add_purchase_order_placeholder": "Purchase order",
+  "purchaseOrders_form_add_purchase_order_sub_info": "Le nom ne doit pas excéder 20 caractères",
+  "purchaseOrders_form_add_purchase_startDate": "Date de début",
+  "purchaseOrders_form_add_purchase_endDate": "Date de fin",
+  "purchaseOrders_form_add_purchase_submit": "Valider",
+  "purchaseOrders_form_add_purchase_cancel": "Annuler",
+  "purchaseOrders_form_add_purchase_submit_REFERENCE_ORDER_success": "Votre référence interne a bien été créée !",
+  "purchaseOrders_form_add_purchase_submit_PURCHASE_ORDER_success": "Votre purchase order a bien été créé !",
+  "purchaseOrders_form_add_purchase_submit_error": "Votre demande n'a pas pu être prise en compte. Merci d'essayer ultérieurement."
```

Comment:
If PU decide to update we have to update here also.
same as for: https://github.com/ovh/manager/pull/6441/files#r809780462.